modular_skyrat/modules/moretraitoritems/code/syndiemirror.dm

```diff
@@ -0,0 +1,81 @@
+/obj/item/hhmirror/syndie
+	name = "Handheld Mirror"
+	desc = "A handheld mirror that allows you to change your looks. Reminds you of old times for some reason..."
+	icon = 'modular_skyrat/master_files/icons/obj/hhmirror.dmi'
+	icon_state = "hhmirror"
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc = "A mirror manufactured by the Syndicate containing barber-nanites that can alter your hair on the spot. Target your head and use it on yourself to activate the nanites."
+	w_class = WEIGHT_CLASS_TINY
+	// How long does it take to change someone's hairstyle?
+	var/haircut_duration = 2 SECONDS
+	// How long does it take to change someone's facial hair style?
+	var/facial_haircut_duration = 2 SECONDS
+
+/obj/item/storage/box/syndie_kit/chameleon/PopulateContents()
+	new /obj/item/clothing/under/chameleon(src)
+	new /obj/item/clothing/suit/chameleon(src)
+	new /obj/item/clothing/gloves/chameleon(src)
+	new /obj/item/clothing/shoes/chameleon(src)
+	new /obj/item/clothing/glasses/chameleon(src)
+	new /obj/item/clothing/head/chameleon(src)
+	new /obj/item/clothing/mask/chameleon(src)
+	new /obj/item/clothing/neck/chameleon(src)
+	new /obj/item/storage/backpack/chameleon(src)
+	new /obj/item/storage/belt/chameleon(src)
+	new /obj/item/radio/headset/chameleon(src)
+	new /obj/item/stamp/chameleon(src)
+	new /obj/item/pda/chameleon(src)
+	new /obj/item/hhmirror/syndie(src)
+	new /obj/item/dyespray(src)
```

Comment:
(Continuing last)
Rather than redo all the contents, couldn't we do
`/obj/item/storage/box/syndie_kit/chameleon/PopulateContents()
. = ..()
new (ourstuff)`
?

Or is that not worth doing; up to a maintainer

Reply:
this WORKS, still maintainer decision if its worth even doing